client/src/components/Pill/Pill.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./Pill.css";
+
+// Function based React Component
+const Pill = props => {
+  // Default Class to apply to Component
+  let classList = `PillContainer`;
+
+  return (
+    <div className={classList}>
+      <h1 className="pillContainer">Topic</h1>
```

Comment:
Can you make the CSS class `pill-container` (CSS Class names should be lowercase and kebab case)